Cabal/Cabal.cabal

```diff
@@ -85,7 +85,7 @@ library
     deepseq    >= 1.3 && < 1.5,
     filepath   >= 1.3 && < 1.5,
     pretty     >= 1.1 && < 1.2,
-    process    >= 1.1.0.1 && < 1.7,
+    process    >= 1.2.1.0 && < 1.7,
```

Comment:
That's a bit a of problem though as GHC 7.4 & 7.6 both bundle process-1.1 and we try hard to support the GHC bundled versions of boot libs.